launcher/src/main/java/org/apache/spark/launcher/CommandBuilderUtils.java

```diff
@@ -260,15 +260,14 @@ static String quoteForBatchScript(String arg) {
         quoted.append('"');
         break;
 
-      case '=':
```

Comment:
Did you run SparkLauncherSuite on Windows? I added this specifically to fix an issue with that path (see 92a9cfb).